src/formatter/helpers/issue_helpers.ts

```diff
@@ -37,6 +37,7 @@ export interface IFormatIssueRequest {
   snippetBuilder: StepDefinitionSnippetBuilder
   testCaseAttempt: ITestCaseAttempt
   supportCodeLibrary: ISupportCodeLibrary
+  printAttachments?: Boolean
```

Comment:
I don't think it matters at runtime but for consistency could we type as `boolean`?